src/components/dialogs/inputs/use-autocomplete-field.js

```diff
@@ -109,20 +109,18 @@ export const useAutocompleteField = ({
 
     useEffect(() => {
         function validate() {
-            const res = validateField(value?.id, validationRef.current);
+            const res = validateField(getLabel(value), validationRef.current);
             setError(res?.errorMsgId);
             return !res.error;
         }
 
-        if (inputForm) {
-            inputForm.addValidation(id ? id : label, validate);
-        }
-    }, [label, validation, inputForm, value, selectedValue, id]);
+        inputForm?.addValidation(id ? id : label, validate);
```

Comment:
you can remplace this `id ? id : label` by `id ?? label`

Reply:
It would not be the same if id === ''
But id || label would do.